rh_ui/security.py

```diff
@@ -0,0 +1,59 @@
+CSP = {
+    'default-src': [
+        "'self'",
+        'https://cdn.ons.gov.uk',
+    ],
+    'font-src': [
+        "'self'",
+        'data:',
+        'https://cdn.ons.gov.uk',
+    ],
+    'script-src': [
+        "'self'",
+        'https://*.googletagmanager.com',
+        'https://cdn.ons.gov.uk',
+    ],
+    'connect-src': [
+        "'self'",
+        'https://cdn.ons.gov.uk',
+        'https://*.google-analytics.com',
+        'https://*.analytics.google.com',
+        'https://*.googletagmanager.com'
+    ],
+    'img-src': [
+        "'self'",
+        'data:',
+        'https://*.google-analytics.com',
+        'https://*.googletagmanager.com',
+        'https://cdn.ons.gov.uk'
+    ],
+}
+
+PERMISSION_POLICY = ('accelerometer=(),autoplay=(),camera=(),display-capture=(),document-domain=(),'
+                     'encrypted-media=(),fullscreen=(),geolocation=(),gyroscope=(),magnetometer=(),'
+                     'microphone=(),midi=(),payment=(),picture-in-picture=(),publickey-credentials-get=(),'
+                     'screen-wake-lock=(),sync-xhr=(self),usb=(),xr-spatial-tracking=()')
+
+DEFAULT_RESPONSE_HEADERS = {
+    'X-Frame-Options': 'DENY',
+    'X-Permitted-Cross-Domain-Policies': 'None',
+    'clear-site-data': '"storage"',
+    'Cross-Origin-Opener-Policy': 'same-origin',
+    'Cross-Origin-Resource-Policy': 'same-site',
+    'Cache-Control': ['no-store', 'max-age=0'],
+    'Server': 'Office For National Statistics',
+}
+
+
+def build_response_headers():
```

Comment:
Why do we need to build our own headers when we're using the Talisman module? Is it to set extra headers that talisman doesn't support? Or am I misunderstanding the purpose/function of Talisman? Either way, if we need both, it would be good to add a few doc strings explaining why.